### PR TITLE
Refs [TASK] [35324] Tạo giao diện cho trang Hot

### PR DESCRIPTION
**Pull request by trinq-2471**
## Related Tickets
- [#Task 35324: Tạo giao diện cho trang Hot](https://edu-redmine.sun-asterisk.vn/issues/35324)
## WHAT
Screenshot from 2021-04-12 11-36-08

## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://github.com/framgia/coding-standards/tree/master/eng/android |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Sun* Redmine working process  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md |<li>- [ ] yes</li>|<li>- [ ] yes</li>  

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

**Comment by phong10101998** [comment]
Thêm phần What và Screen Shot nha a

File: app/src/main/res/layout/fragment_hot.xml [hunk 1]
```diff
@@ -1,7 +1,75 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/dp_10"
+            android:layout_below="@+id/btnUpComming"
+            android:background="@color/blue_dark">
+        </androidx.recyclerview.widget.RecyclerView>
+
+    </RelativeLayout>
+
+</androidx.swiperefreshlayout.widget.SwipeRefreshLayout>
```
Comment (phong10101998) [inline app/src/main/res/layout/fragment_hot.xml]:
```suggestion
        </androidx.recyclerview.widget.RecyclerView>
    </RelativeLayout>
</androidx.swiperefreshlayout.widget.SwipeRefreshLayout>
```

File: app/src/main/res/layout/item_hot_movie.xml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginBottom="@dimen/dp_20"
+    android:background="@color/blue_dark">
+
+    <ImageView
+        android:id="@+id/imgHotMovie"
+        android:layout_width="@dimen/dp_200"
+        android:layout_height="@dimen/dp_260"
+        android:src="@drawable/ic_launcher_foreground"
```
Comment (phong10101998) [inline app/src/main/res/layout/item_hot_movie.xml]: Không set cứng ảnh

File: app/src/main/res/values/dimes.xml [hunk 3]
```diff
@@ -1,10 +1,17 @@
...
     <dimen name="dp_15">15dp</dimen>
+    <dimen name="dp_30">30dp</dimen>
+    <dimen name="dp_35">35dp</dimen>
     <dimen name="dp_40">40dp</dimen>
     <dimen name="dp_50">50dp</dimen>
+    <dimen name="dp_200">200dp</dimen>
+    <dimen name="dp_260">260dp</dimen>
     <dimen name="sp_20">20sp</dimen>
+    <dimen name="sp_12">12sp</dimen>
 </resources>
```
Comment (phong10101998) [inline app/src/main/res/values/dimes.xml]: Sắp xếp lại , phân dp và sp ra

File: app/src/main/res/layout/fragment_hot.xml [hunk 4]
```diff
@@ -1,7 +1,75 @@
...
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    tools:context=".ui.hot.HotFragment">
-</FrameLayout>
+    android:layout_marginTop="@dimen/dp_10"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
```
Comment (phong10101998) [inline app/src/main/res/layout/fragment_hot.xml]: Tập dùng constrainLayout đi anh

File: app/src/main/res/layout/item_hot_movie.xml [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/dp_10"
+        android:text="@string/favorite"
+        android:textSize="@dimen/sp_20"
+        android:textColor="@color/white"
+        android:gravity="center"
+        android:layout_below="@+id/rateBarHotMovie">
+    </TextView>
+
+</RelativeLayout>
```
Comment (phong10101998) [inline app/src/main/res/layout/item_hot_movie.xml]:
```suggestion
    </TextView>
</RelativeLayout>
```

**Review by phong10101998 — APPROVED** [review]

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**